src/Tools/babylon.tools.ts

```diff
@@ -698,6 +698,12 @@
             return false;
         }
 
+        public static getClassName(obj): string {

```

Comment:
This is the reason why the Actions Builder uses the babylon.max.js file instead of the minified version :(